src/OpenTelemetry.Instrumentation.AWS/Implementation/Tracing/AWSTraceSpan.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using Amazon.Runtime.Telemetry;
+using Amazon.Runtime.Telemetry.Tracing;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Instrumentation.AWS.Implementation.Tracing;
+
+internal class AWSTraceSpan : TraceSpan
```

Comment:
Make `AWSTraceSpan`, `AWSTracer` and `AWSTracerProvider` `sealed` to aid with JIT optimisations?